infrastructure/modules/fhir-api/main.tf

```diff
@@ -51,6 +51,23 @@ resource "aws_iam_policy" "fhir_api_can_access_fhir_api_db_secret" {
   })
 }
 
+resource "aws_iam_policy" "fhir_api_can_access_fhir_api_superuser_password" {
+  name        = "${var.account_name}-fhir-api-can-access-fhir-superuser-password"
+  description = "Allows ECS to access the pbkdf2 encoded default superuser password"
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Action = "secretsmanager:GetSecretValue",
+        Effect = "Allow"
+        Resource = [
+          var.fhir_api_superuser_password_arn
+        ]
+      }
+    ]
+  })
+}
```

Comment:
This needs a corresponding role/policy attachment, which associates the policy to the execution role.

```hcl
resource "aws_iam_role_policy_attachment" "fhir_api_can_access_fhir_api_superuser_secret_attachment" {
  role       = aws_iam_role.fhir_api_execution_role.name
  policy_arn = aws_iam_policy.fhir_api_can_access_fhir_api_superuser_password.arn
}
```